sdk/keyvault/tests.yml

```diff
@@ -8,12 +8,18 @@ jobs:
       ServiceDirectory: keyvault
       EnvVars:
         AZURE_TEST_MODE: RECORD
-        AZURE_TEST_HTTP_CLIENTS: ALL
-        AZURE_KEYVAULT_TEST_CERTIFICATE_SERVICE_VERSIONS: ALL
-        AZURE_KEYVAULT_TEST_KEYS_SERVICE_VERSIONS: ALL
-        AZURE_KEYVAULT_TEST_CRYPTOGRAPHY_SERVICE_VERSIONS: ALL
-        AZURE_KEYVAULT_TEST_SECRETS_SERVICE_VERSIONS: ALL
         ARM_CLIENTID: $(aad-azure-sdk-test-client-id)
         ARM_CLIENTKEY: $(aad-azure-sdk-test-client-secret)
         AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
         AZURE_KEYVAULT_ENDPOINT: $(AZURE_KEYVAULT_ENDPOINT)
+      Matrix:
+        Linux - Java 8:
+            OSName: 'Linux'
+            OSVmImage: 'ubuntu-16.04'
+            JavaVersion: '1.8'
+            DisplayName: 'Linux - Java 8'
+            AZURE_TEST_HTTP_CLIENTS: NettyAsyncHttpClient
+            AZURE_KEYVAULT_TEST_CERTIFICATE_SERVICE_VERSIONS: 7.0
```

Comment:
Assuming this is to be set to latest version, 
latest version is this: 7.1-preview

Reply:
The tests are failing. Try to see whether we need to add quote for env value